### DISCUSSION TITLE
15/05/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#149] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimento

**Comment by Pamela198713** [comment]
Ayer acomode el PPT lo descarge y lo subí a la pag de entrega de actividad.
Hoy tengo clases de emprendedurismo veré qué hay q agregar a documentación. Actualizar.
El tiempo y todas las actividades del ispc que hay que hacer.

**Comment by rpgrca** [comment]
Ayer terminamos con lo de emprendedurismo, también armé unos videos sobre el producto web. Hoy estuve en el dentista bien temprano así que entre dormir poco y la anestesia en la boca no puedo hablar así que tal vez grabe los videos que faltan a la noche recién. Voy a retomar la parte de Android en cuanto consiga un poco de tiempo.

**Comment by fmdcba** [comment]
Ayer estuve en la clase y para la entrega de emprendedurismo atento a lo que faltaba. Hoy voy a estar en clases de tecnología y ver que puedo ir sumando a ese trabajo.
